feg/gateway/services/s6a_proxy/servicers/s6a_definitions.go

```diff
@@ -379,4 +380,5 @@ type ULR struct {
 	VisitedPLMNID    datatype.Unsigned32       `avp:"Visited-PLMN-Id"`
 	RATType          datatype.Unsigned32       `avp:"RAT-Type"`
 	ULRFlags         datatype.Unsigned32       `avp:"ULR-Flags"`
+	SupportedFeatures           []SupportedFeatures         `avp:"Supported-Features"`
```

Comment:
maybe run go fmt? the spacing looks weird

Reply:
yep, I think I forgot to amend after running make pre commit 😅